### PR TITLE
vttest 20220215

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
flaky run on linux, give another shot

**Comment by SMillerDev** [comment]
I guess the connections need to be done with Homebrew curl since all are failing on Linux

File: Formula/vttest.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Vttest < Formula
   desc "Test compatibility of VT100-compatible terminals"
   homepage "https://invisible-island.net/vttest/"
-  url "https://invisible-mirror.net/archives/vttest/vttest-20210210.tgz"
-  sha256 "0f98a2e305982915f1520984c3e8698e3acd508ee210711528c89f5a7ea7f046"
+  url "https://invisible-mirror.net/archives/vttest/vttest-20220215.tgz"
```
Comment (issyl0) [inline Formula/vttest.rb:4]:
```suggestion
  url "https://invisible-mirror.net/archives/vttest/vttest-20220215.tgz", using: :homebrew_curl
```

**Review by issyl0 — CHANGES_REQUESTED** [review]

**Review by issyl0 — APPROVED** [review]
Thanks @Porkepix and @SMillerDev!

**Pull request closed without merging**